analyses/cellranger-analysis/util/run_cellranger.py

```diff
@@ -0,0 +1,154 @@
+parser.add_argument('--include_introns', help='Optional. Set to false to exclude intronic reads in count. Including introns in analysis is recommended to maximize sensitivity. Refer to the guide on including introns for gene expression analysis for comprehensive recommendations. Default: true (in Cell Ranger v7.0 and later, intronic reads are counted by default)', action="store_true")
+parser.add_argument('--chemistry', help='Optional. Assay configuration. By default the assay configuration is detected automatically, which is the recommended mode. You should only specify chemistry if there is an error in automatic detection or for some special cases.', type=str)
+parser.add_argument('--check_library_compatibility', help='Optional. This option allows users to disable the check that evaluates 10x Barcode overlap between libraries when multiple libraries are specified (e.g., Gene Expression + Antibody Capture). Setting this option to false will disable the check across all library combinations. We recommend running this check (default), however if the pipeline errors out, users can bypass the check to generate outputs for troubleshooting (true, false; default: true).', type=str)
+parser.add_argument('--nosecondary', help='Optional. Adding --nosecondary to the command disables the generation of secondary analysis results, such as clustering and dimensionality reduction (tSNE/UMAP) and the .cloupe output file. This flag does not require a true or false value to be specified.', action="store_true")
+parser.add_argument('--min_crispr_umi', help='Optional. Set the minimum number of CRISPR guide RNA UMIs required for protospacer detection. Default: 3 UMIs (in Cell Ranger v8.0 and later). If a lower or higher sensitivity is desired for detection, this value can be customized according to specific experimental needs. Applicable only to datasets that include a CRISPR Guide Capture library.', type=int)
+parser.add_argument('--file', help='Optional. Tsv file containing the following	header and corresponding info (#ID SAMPLE FASTQ) for batch submission.', type=argparse.FileType('r'))
+
+
+#Converts argument strings to objects and assigns them as attributes of the namespace; e.g. --id -> args.id
+args = parser.parse_args()
+
+#Accessing reference genome based on requested short name of reference genome
+if args.transcriptome == "GRCh38":
+	args.transcriptome = "./ReferenceGenomes/homo_sapiens/GRCh38/downloads/refdata-gex-GRCh38-2024-A/"
+elif args.transcriptome == "GRCm39":
+	args.transcriptome = "./ReferenceGenomes/mus_musculus/GRCm39/downloads/refdata-gex-GRCm39-2024-A/"
+elif args.transcriptome == "GRCh38ANDGRCm39":
+	args.transcriptome = "./ReferenceGenomes/homo_sapiens_mus_musculus/GRCh38_GRCm39/downloads/refdata-gex-GRCh38_and_GRCm39-2024-A/"
+elif args.transcriptome == "GRCh38_GFP_tdTomato":
+	args.transcriptome = "./cab/Automation/REF/Homo_sapiens/Gencode_GFP/r31/CellRanger-index/7.0.1/"
```

Comment:
@sharonfreshour These hardcoded file paths have been removed, FYI. There is an explanation in the `README` file of how to access these.